dashboard/package.json

```diff
@@ -17,11 +17,11 @@
     "@types/moxios": "^0.4.11",
     "@types/pako": "^1.0.1",
     "@types/qs": "^6.9.6",
-    "@types/react": "^17.0.5",
+    "@types/react": "^17.0.6",
```

Comment:
Not sure why this file has come into the PR

Reply:
Since the branches are not in perfect sync, some changes are also included. No worries, once we merge them into my branch and, later, into the Kubeapps main branch, they will disappear.